libfwupd/fwupd-request.h

```diff
@@ -53,12 +54,33 @@ typedef enum {
 /**
  * FWUPD_REQUEST_ID_PRESS_UNLOCK:
  *
- * The user needs to press unlock on the device.
+ * The user needs to press unlock on the device to continue, e.g.
+ * "Press unlock on the device to continue the update process."
  *
  * Since 1.6.2
  */
 #define FWUPD_REQUEST_ID_PRESS_UNLOCK "org.freedesktop.fwupd.request.press-unlock"
 
+/**
+ * FWUPD_REQUEST_ID_REMOVE_DEVICE:
+ *
+ * The user needs to remove the device to complete the update, e.g.
+ * "The update will continue when the device USB cable has been unplugged."
+ *
+ * Since 1.8.6
+ */
+#define FWUPD_REQUEST_ID_REMOVE_DEVICE "org.freedesktop.fwupd.request.remove-device"
+
+/**
+ * FWUPD_REQUEST_ID_DO_NOT_POWER_OFF:
+ *
+ * Show the user a message not to unplug the machine from the AC power, e.g.
+ * "Do not turn off your computer or remove the AC adaptor while the update is in progress."
```

Comment:
Knowing some vendors have black screens as part of their update process how about instead "Until the update is complete".

Reply:
`Do not turn off your computer or remove the AC adaptor until you are sure the update is complete.` -- maybe?

Reply:
That's good

Reply:
Fixed.